apps/easy-messe/components/Prefooter/QuoteMasses.tsx

```diff
@@ -0,0 +1,155 @@
+import { theme } from "@easy-messe/libs/theme";
+import LeftIcon from '@iconify-icons/fluent/chevron-left-20-regular';
+import RightIcon from '@iconify-icons/fluent/chevron-right-20-regular';
+import { Icon } from "@iconify/react";
+import { Box, IconButton, Typography } from "@mui/material";
+import { useState } from "react";
+
+interface quoteMasses {
```

Comment:
- [ ] interfaces are in PascalCase
- [ ] interfaces are to be named in the singular

```suggestion
interface QuoteMass {
```

btw, what is QuoteMass?
what does it mean?

Reply:
Quote here stand for a short message provide by a man of God or woman or famous person in God domain related to a typical topic  of faith.

Reply:
In our case, we have the mass which is the main subject we process.